analytics/analytics.go

```diff
@@ -85,13 +85,26 @@ func SendAnonymizedAnalytics(buildRunResults models.BuildRunResultsModel) error
 		runtime       time.Duration
 		stepAnalytics []StepAnalytics
 	)
+
+	stepInputWhitelist := map[string]bool{
```

Comment:
pls also filter for the step, we only want to collect `xcode-test` inputs, something like:
```
stepInputWhitelist := map[string]map[string]bool{
  "xcode-test": map[string]bool{
    "simulator_device": true,
....
  }
}
```